config/values/bursaries.yml

```diff
@@ -0,0 +1,25 @@
+bursaries:
+  SKE: £175
+  veteran:
+    full-amount: £40,000
+    installment-amount: £20,000
+## implemented above this line
+  undergraduate: £9,000
+  postgraduate:
```

Comment:
could these be changed to be in alphabetical order? will make it much easier to update :-)